pkg/server/message_test.go

```diff
@@ -93,7 +93,7 @@ func Test_fuzzyTranslate(t *testing.T) {
 			err := translateSrv.fuzzyTranslate(ctx, allMessages)
 			require.NoError(t, err)
 
-			// Check that untranslated messages have been translated and marked as fuzzy for all translations.
+			// Check that untranslated translation have been translated and marked as fuzzy for all translations.
```

Comment:
// Check that untranslated messages have been translated and marked as fuzzy for all translations.